save-backend/src/main/kotlin/com/saveourtool/save/backend/controllers/ProjectController.kt

```diff
@@ -95,6 +96,23 @@ class ProjectController(
             projectPermissionEvaluator.hasPermission(authentication, it, Permission.READ)
         }
 
+    @PostMapping("/not-deleted-filter")
+    @PreAuthorize("permitAll()")
+    @Operation(
+        method = "POST",
+        summary = "Get non-deleted projects.",
+        description = "Get non-deleted projects, available for current user.",
+    )
+    @ApiResponse(responseCode = "200", description = "Successfully fetched non-deleted projects.")
+    fun getNotDeletedProjectsWithFilters(
+        @RequestBody(required = false) projectFilters: ProjectFilters?,
+        authentication: Authentication?,
+    ): Flux<Project> = projectService.getNotDeletedProjectsWithFilter(projectFilters?.name)
```

Comment:
I think its better to pass filters into service and unpack values there if required

Reply:
Fixed